tests/queries/0_stateless/02346_gin_index_definition.sql

```diff
@@ -0,0 +1,163 @@
+SET allow_experimental_full_text_index = 1;
+DROP TABLE IF EXISTS tab;
+
+SELECT 'GIN index only literals. (backward compatibility)';
```

Comment:
No need to retain backward compat, it is an experimental feature. Lets remove all C++ code for that.